cristin-import/src/main/java/no/unit/nva/cristin/lambda/ApplicationConstants.java

```diff
@@ -0,0 +1,10 @@
+package no.unit.nva.cristin.lambda;
+
+public final class ApplicationConstants {
+
+    public static final String EVENT_BUS_NAME = "default";
```

Comment:
These will be filled in later and the values are going to be set by env variables